src/components/timeseries/TimeSeries.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TIMESERIES_DASHBOARD_URL } from '../../constants';
+import ErrorMessageWithIcon from '../error/ErrorMessageWithIcon';
+
+import ScreenStatusMessage from './ScreenStatusMessage';
+
+class TimeSeries extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            maintenanceMode: this.props.maintenanceMode
+        };
+    }
+
+    render() {
+        return (
+            (!this.state.maintenanceMode) ?
+                <div>
+                    <ScreenStatusMessage maintenanceMode={this.state.maintenanceMode} />
```

Comment:
I feel you are not using `maintenanceMode` props in the component, right?

Reply:
Right. I will remove on the next commit